v2/azure-arm.yaml

```diff
@@ -1587,6 +1599,11 @@ objectModelConfiguration:
       VirtualNetworkRule:
         Id:
           $armReference: true
+  kubernetesconfiguration:
+    2022-11-01:
```

Comment:
As mentioned above, 2023-05-01 is a newer API version than this -- should we use that instead?